error/src/lib.rs

```diff
@@ -271,11 +271,13 @@ impl CaliptraError {
     pub const RUNTIME_ECDSA_VERIF_FAILED: CaliptraError = CaliptraError::new_const(0x000E0004);
 
     /// FMC Errors
-    pub const FMC_RT_ALIAS_DERIVE_FAILURE: CaliptraError = CaliptraError::new_const(0x000F0001);
-    pub const FMC_HANDOFF_INVALID_PARAM: CaliptraError = CaliptraError::new_const(0x000F0002);
-    pub const FMC_GLOBAL_NMI: CaliptraError = CaliptraError::new_const(0x000F0003);
-    pub const FMC_GLOBAL_EXCEPTION: CaliptraError = CaliptraError::new_const(0x000F0004);
-    pub const FMC_GLOBAL_PANIC: CaliptraError = CaliptraError::new_const(0x000F0005);
+    pub const FMC_GLOBAL_NMI: CaliptraError = CaliptraError::new_const(0x000F0001);
+    pub const FMC_GLOBAL_EXCEPTION: CaliptraError = CaliptraError::new_const(0x000F0002);
+    pub const FMC_GLOBAL_PANIC: CaliptraError = CaliptraError::new_const(0x000F0003);
+    pub const FMC_HANDOFF_INVALID_PARAM: CaliptraError = CaliptraError::new_const(0x000F0004);
+    pub const FMC_RT_ALIAS_DERIVE_FAILURE: CaliptraError = CaliptraError::new_const(0x000F0005);
+    pub const FMC_RT_ALIAS_CERT_VERIFY: CaliptraError = CaliptraError::new_const(0x000F0005);
```

Comment:
0x000F0005 is used for both FMC_RT_ALIAS_DERIVE_FAILURE and FMC_RT_ALIAS_CERT_VERIFY.

Reply:
good catch, brother!